File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/entity/Anonymous.java
package cn.evole.onebot.sdk.entity;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

@Data
public class Anonymous {

    @SerializedName( "id")
    private long id;

    @SerializedName( "name")
    private String name;

    @SerializedName( "flag")
    private String flag;

}

File: overflow-core/src/main/kotlin/cn/evolvefield/onebot/sdk/entity/Anonymous.kt
package cn.evolvefield.onebot.sdk.entity

import com.google.gson.annotations.SerializedName

class Anonymous {
    @SerializedName("id")
    var id = 0L
    @SerializedName("name")
    var name: String = ""
    @SerializedName("flag")
    var flag: String = ""
}
